### DISCUSSION TITLE
AttributeError

### DISCUSSION THREAD

**Discussion by againmain1** [JonathanFly/bark#84] [Q&A]
Hey!

I got stuck with this error when I'm trying to launch the "LAUNCH_already_installed_bark_infinity_windows.bat" file:
Has anybody faced the same problem? 
![image](https://github.com/JonathanFly/bark/assets/60272309/de69d8db-9785-4557-badb-7d30af1bf743)

**Comment by JonathanFly** [comment]
Should fix fixed now if you do a `git pull`
